src/app/_components/most-popular-section/most-popular.tsx

```
@@ -0,0 +1,63 @@
+
+  useEffect(() => {
+    const occasion = occasionsList.find((o) => o.name === activeTab);
+    if (occasion) {
+      getOccasionById(occasion.id).then((res) => {
+        setData(res?.occasion);
+      });
+    }
+  }, [activeTab]);
+
+  return (
+    <div className="p-6">
```

ESLint

[error] className value must use cn() function. Example: className={cn("...")}

(local-rules/require-cn-classname)